### PR TITLE
[#177060967] bugfixing inputcard

### PR DESCRIPTION
**Pull request by vito80ba**
This PR is aimed to fix several cosmetic & interaction fixes.

#### List of Changes

- Inputcard typescript: various changes
- inputcard pug: various changes
- style: variuos changes

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/inputcard.ts [hunk 1]
```diff
@@ -62,6 +76,25 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
+  // Custom validator for CVV
+  function checkCvvSize(): void {
+    const size = (creditcardformSecurecode && creditcardformSecurecode.getAttribute('data-validator-size')) || '3';
```
Comment (GiovanniMancini) [inline src/inputcard.ts:81]: I can't understand how the right size of the security code is computed as it's known to depend on country and circuit

**Review by GiovanniMancini — APPROVED** [review]

File: src/inputcard.ts (hunk 1 above)
Comment (vito80ba) [reply src/inputcard.ts:81]: Every time the credit number field is modified, a correct size of cc is set with a data-attributed on the secure code field, cause the credit-card number (circuit) decided how long could be the secure code (CID in Amex, CVV for the other one)

**Review by pasqualespica — CHANGES_REQUESTED** [review]
Great work , but I have a doubt on `Titolare Carta`'s check

File: src/inputcard.ts [hunk 2]
```diff
@@ -238,61 +270,72 @@ document.addEventListener('DOMContentLoaded', () => {
...
-    const regpatternAtLeastOneSpace = new RegExp('(\\w.+\\s).+', 'i');
-
-    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
-    regpatternCharacther.test(inputel.value) === true && regpatternAtLeastOneSpace.test(inputel.value)
-      ? toggleValid(inputel, true)
-      : toggleValid(inputel, false);
+  creditcardformName?.addEventListener('keyup', evt => {
+    const inputel: HTMLInputElement = evt?.target as HTMLInputElement;
+
+    const regpattern = new RegExp(/^[a-zA-Z]+[\s']+([a-zA-Z]+[\s']*){1,}$/);
```
[the remark this answers is not in the document]
Comment (vito80ba) [reply src/inputcard.ts:276]: Yes, imho. Maybe we should check this with PM's guys

**Pull request merged**